package.json

```diff
@@ -48,6 +48,7 @@
     "@aws-cdk/aws-ec2": "1.37.0",
     "@aws-cdk/aws-ecr": "1.37.0",
     "@aws-cdk/aws-ecs": "1.37.0",
+    "@aws-cdk/aws-elasticache": "1.37.0",
```

Comment:
need to add this to also the service package json ad a peer dep

Reply:
Done